src/main/java/org/folio/circulation/domain/Item.java

```diff
@@ -143,7 +144,12 @@ public String getInstanceId() {
   }
 
   public String getCallNumber() {
-    return getProperty(holdingRepresentation, "callNumber");
+    String propertyName = "callNumber";
+    String callNumber = getProperty(holdingRepresentation, propertyName);
+
+    return StringUtils.isBlank(callNumber)
```

Comment:
Can you help me understand where this logic comes from? Is it defined in a story?

My understanding was that how this would work was still being discussed by the relevant SIGs.

Reply:
Having just read the story, it specifically states `From holding for now, Just want to pull the simple <callNumber> field`

Reply:
@marcjohnson-kint Yes, but it is empty value to represent an item in the `holdingRepresentation` request. So in order to get at least some callNumber value, I get it from the `itemRepresentation`.

Reply:
I don't think I understand.

The story specifically states that we take the `callNumber` from the holdings record.

Are you saying that to test this, there is a need for an item with a holding that has a call number?

Reply:
I mean that when we get an item from a request (circ-storage) it only has an identifier and a barcode. 
I added this field to a representation of a request that will be written to the circ-storage. And now I'm pulling it from out of `itemRepresentation`.

Reply:
> I added this field to a representation of a request that will be written to the circ-storage. And now I'm pulling it from out of itemRepresentation.

We only write properties of other records to storage if they are needed for specific reasons. The item properties stored with a request are only present for sorting and filtering.

Is the call number necessary for searching or sorting? Or was it hoped that storing meant working around needed to derive it during the report?

Reply:
>    Or was it hoped that storing meant working around needed to derive it during the report?
Yes, only for this was done.

Reply:
I don't think we should be introducing unnecessary storing of item properties with the request.

Why are we not following the standard approach of fetched the related records and deriving this information during the creation of the report?

Reply:
Rollback commit: 5d81e11 / Add callNumber to request representation
I will try to follow the standard approach of fetched the related records

Reply:
Thank you. I am trying to limit the impact of this workaround as much as possible.

If this becomes a challenge, please let me know and I can try to help. Or we can decide if it makes sense to spread the impact and increase the rework needed later.